src/confighttp.cpp

```diff
@@ -32,6 +32,7 @@
 #include "file_handler.h"
 #include "globals.h"
 #include "httpcommon.h"
+#include "jwt-cpp/jwt.h"
```

Comment:
```suggestion
#include <jwt-cpp/jwt.h>
```

This should probably also be up with the simple web server includes.